### PR TITLE
Link privacy policy

### LINKED ISSUE #377: Link privacy policy
## Kind (Required)

- Improvement

 ## Overview (Required)

- There is a link to the privacy policy in AboutThisApp, but it's not working. Please change so that you can transition to the link from there.

 ## Links	

- http://www.association.droidkaigi.jp/privacy.html

### PR DESCRIPTION
**Pull request by k-shinn**
## Issue
- close #377 

## Overview (Required)
- Add link to privacy policy with `AppActions#getShowChromeCustomTabs`

## Links
-

### CODE REVIEW

File: uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt [hunk 1]
```diff
@@ -132,6 +132,9 @@ fun AppContent(
                     },
                     onStaffClick = { staff ->
                         actions.showChromeCustomTabs(staff.url)
+                    },
+                    onPrivacyPolicyClick = {
+                        actions.showChromeCustomTabs(it)
```
Comment (takahirom) [inline uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt:137]: 👍

File: uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt [hunk 2]
```diff
@@ -106,7 +109,7 @@ fun OtherScreen(
                     modifier = Modifier.fillMaxHeight()
                 ) {
                     FadeThrough(targetState = selectedTab) { selectedTab ->
-                        BackdropFrontLayerContent(selectedTab, onContributorClick, onStaffClick)
+                        BackdropFrontLayerContent(selectedTab, onContributorClick, onStaffClick, onPrivacyPolicyClick)
```
Comment (takahirom) [inline uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt:112]: This seems to cause CI to fail, so can I add a newline? 🙏 

```
Error on line: 112, column: 1
  Exceeded max line length (100)
```

Comment (k-shinn) [reply uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt:112]: Sorry! I'll fix it! 🙇

**Comment by takahirom** [comment]
Thank you! LGTM!
Either is fine, but it seems that the email address registered in Git commit is different from the email address registered in GitHub, so if you want to register, you may want to rebase.

```
git config --global user.email your@email.com  
```
https://github.com/DroidKaigi/conference-app-2021/graphs/contributors

**Comment by k-shinn** [comment]
@takahirom 
Thanks for letting me know! 🙇 
I'll give it a shot and get back to you later!

**Comment by k-shinn** [comment]
@takahirom 
I fixed problems so please check it.
Thank you 🙇‍♂️

**Review by takahirom — APPROVED** [review]
Thank you for cooperation!

**Pull request merged**